addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/blinds1.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="rfxcom"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="blinds1">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+			<bridge-type-ref id="RFXtrx433" />
+			<bridge-type-ref id="RFXrec433" />
+		</supported-bridge-type-refs>
+
+		<label>RFXCOM Blind1 Sensor</label>
+		<description>Generic thing type for Blinds1 devices.</description>
+
+		<channels>
+			<channel id="command" typeId="command" />
+			<channel id="shutter" typeId="shutter" />
+			<channel id="signalLevel" typeId="system.signal-strength" />
+			<channel id="batteryLevel" typeId="batterylevel" />
```

Comment:
Note that there has been recently added a `system.battery-level` as well, see https://github.com/eclipse/smarthome/blob/master/docs/documentation/development/bindings/thing-definition.md#channels. Could you change your implementation to use it?